shared/constants/types/wallets.js

```diff
@@ -212,12 +212,17 @@ export type Payment = I.RecordOf<_Payment>
 
 export type Currency = I.RecordOf<_LocalCurrency>
 
+export type AccountMode = StellarRPCTypes.AccountMode
+export const accountModeNone = StellarRPCTypes.commonAccountMode.none
+
 export type _Account = {
   accountID: AccountID,
   balanceDescription: string,
   displayCurrency: Currency,
   isDefault: boolean,
   name: string,
+  accountMode: AccountMode,
```

Comment:
why is this plumbed through?

Reply:
it's not strictly necessary for this effort, but i was thinking about https://keybase.atlassian.net/browse/CORE-9900. exposing accountMode here i think would enable the frontend to drop a whole bunch of machinery for tracking mobileOnly as a separate construct. i'm happy to drop it if you don't think it's helpful, but the data is there in the RPC.

Reply:
lets drop it. we prefer to not plumb things through that don't do anything. if we need it later we can add it